### DISCUSSION TITLE
Recommended way to install Virtualbox on MacOS Big Sur

### DISCUSSION THREAD

**Discussion by RyanSept** [actions/virtual-environments#4787] [Installed software]
Attempting to manually install Virtualbox on the Big Sur image runs into a similar kext problem cited on issue #429.

On the other hand #2999 says adding Virtualbox is not a priority but the workarounds suggested aren't functional and require either disabling of SIP entirely or consenting via the GUI. 

Since we're unable to install it could it at least be prepackaged with the image? If not, could you disable SIP?

**Comment by al-cheb** [comment]
Currently you can use macOS Catalina with pre-installed VirtualBox without any issue.
Since we're unable to install it could it at least be prepackaged with the image? - Blockers https://github.com/actions/virtual-environments/pull/4010#issuecomment-947607896
If not, could you disable SIP? - Nope, https://github.com/actions/virtual-environments/issues/650

**Reply by RyanSept** [reply]
Thanks! Hadn't seen #4010. We're currently using macOS Catalina and needed to run with Big Sur too. Guess we'll just have to wait for the fix for the driver issue mentioned on that PR.
